project_forecast_line_priority/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2024 Therp BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Project Forecast Line Priority",
+    "summary": "Project Forecast Line dates according to task priority",
+    "version": "14.0.1.0.0",
+    "author": "Therp BV, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Project",
+    "website": "https://github.com/OCA/project",
+    "depends": ["project_forecast_line_deadline", "project_task_add_very_high"],
+    "data": ["data/ir_actions_server.xml", "views/res_config_settings.xml"],
+    "installable": True,
+}
```

Comment:
@ntsirintanis can you add `"development_status": "Alpha"` here, so that we don't get the failing checks on us AFAICS defaulting to "Beta" while dependency is in "alpha"? Also for the other modules BTW.